File: Python/UberVan.py
from car import Car

class UberVan(Car):
    """
    UberVan Class
    """
    typeCarAccepted  = []
    seatsMaterial    = []
    def __init__(self,license,driver,typeCarAccepted,seatsMaterial):
        super.__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String,ArrayList<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;


    public UberBlack(String licence,Account driver,Map<String,ArrayList<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(licence, driver);
        this.typeCarAccepted =typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
        
        
    }
    


}